pelux.xml

```diff
@@ -23,7 +23,7 @@
 
   <project remote="github"
            revision="28f52b1451c6bebe46224b81dc6849babf286ce1"
-           upstream="pyro"
+           upstream="master"
```

Comment:
Why this change when bumping meta-pelux?

Reply:
I didn't realize until after I had made the PR that another PR bumped the entire internal layers, and also updated the "upstream" branch name. It is a typo on master.